Tests/ConfidenceTests/ConfidenceValueTests.swift

```diff
@@ -119,6 +125,10 @@ final class ConfidenceConfidenceValueTests: XCTestCase {
         let encoder = JSONEncoder()
         encoder.outputFormatting = .sortedKeys
         let resultString = String(data: try encoder.encode(value), encoding: .utf8)
+
+        let isoFormatter = ISO8601DateFormatter()
+        isoFormatter.timeZone = TimeZone.current
+        let expectedSerializedTimestamp = isoFormatter.string(from: date)
```

Comment:
Computing this value in Stockholm, for example, yields: `2024-04-05T22:00:00+002`

Reply:
After further deliberations we opted to send UTC time from clients (this is most likely the case in other SDKs as well), for simplicity